ui/components.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Commit message: Please add a link to https://ui.shadcn.com/ and provide a rationale why this was chosen over other component frameworks.

Reply:
Missing final newline in file.